### PR TITLE
fix redirected url. Also fix typo in comment.

### PR DESCRIPTION
**Pull request by jonathanbratt**

### CODE REVIEW

File: README.Rmd [hunk 1]
```diff
@@ -16,7 +16,7 @@ knitr::opts_chunk$set(
 # wordpiece
 
 <!-- badges: start -->
-[![Lifecycle: maturing](https://img.shields.io/badge/lifecycle-maturing-blue.svg)](https://www.tidyverse.org/lifecycle/#maturing)
+[![Lifecycle: maturing](https://img.shields.io/badge/lifecycle-experimental-orange.svg)](https://lifecycle.r-lib.org/articles/stages.html#experimental)
```
Comment (jonthegeek) [inline README.Rmd:19]:
```suggestion
[![Lifecycle: experimental](https://img.shields.io/badge/lifecycle-experimental-orange.svg)](https://lifecycle.r-lib.org/articles/stages.html#experimental)
```

**Review by jonthegeek — CHANGES_REQUESTED** [review]
One small tweak (maturing --> experimental in the words). Be sure to reknit the md.

File: README.Rmd (hunk 1 above)
Comment (jonathanbratt) [reply README.Rmd]: fixed locally and pushed

**Review by jonthegeek — APPROVED** [review]
Looks good!

**Pull request merged**